### ISSUE TITLE
Possible collaboration with similar project (Gitchain)

### ISSUE THREAD

**Issue by ghost** [cjb/GitTorrent#80]
This project seems a lot similar to Gitchain (https://github.com/gitchain/gitchain) and the premise is the same, decentralized p2p git repos using DHT technology, (yours focusing on bittorrent and gitchain focusing on Bitcoin, namecoin and other blockchain software. Maybe you could contact each other and collaborate to get this thing off the ground because im really enthusiastic about it and the technologies are quite similar.

**Comment by cjb** [comment]
I was one of the GitChain Kickstarter backers :) It hasn't been touched in three years, though, so I don't think there's any collaboration possible now. You're right the tech is similar, with GitChain more interested in having nodes be miners etc.

**Comment by lukelane124** [comment]
sounds familiar...
What version of npm were you using when this project was "up to date?"

**Comment by rom1504** [comment]
What sounds familiar ? Projects were development have stopped are the
majority of projects. (In GitHub and elsewhere)

**Comment by lukelane124** [comment]
Obviously, it's been a minute since I made that comment, but irc I was talking about Kickstarter code projects never really getting off the ground. However, your comment definitely applies here, projects are quick to be abandoned.
